src/components/pages/Configuration.tsx

```diff
@@ -0,0 +1,283 @@
+
+
+const Configuration: RouteComponent = () => {
+    const {
+        getInstanceElementMetadata,
+        setInstanceElementMetadata
+    } = useContext(SourceApplicationContext)
+    const {
+        completed,
+        setCompleted,
+        resetConfigurationContext
+    } = useContext(ConfigurationContext)
+    const {setEditCollectionAbsoluteKey} = useContext(EditingContext)
+    const {t} = useTranslation('translations', {keyPrefix: 'pages.configuration'});
+    const history = useHistory();
+    const {
+        selectedMetadata,
+        setSelectedMetadata,
+        existingIntegration,
+        configuration,
+        setConfiguration,
+        resetIntegrationContext
+    } = useContext(IntegrationContext)
+    const [active, setActive] = useState<boolean>(existingIntegration?.activeConfigurationId === configuration?.id)
+    const [showAlert, setShowAlert] = React.useState<boolean>(false)
+    const [alertContent, setAlertContent] = React.useState<IAlertContent>(defaultAlert)
+    const [collectionReferencesInEditContext, setCollectionReferencesInEditContext] = useState<string[]>([])
+
+    if (!existingIntegration) {
+        history.push('/')
+    }
+    const methods = useForm<IConfiguration>({
+        mode: 'onChange',
+        defaultValues: {
+            integrationId: Number(existingIntegration?.id),
+            integrationMetadataId: Number(selectedMetadata?.id),
+            completed: configuration ? configuration.completed : false,
+            comment: configuration?.comment,
+        }
+    });
+
+    const handleClose = (event: React.SyntheticEvent | Event, reason?: string) => {
+        if (reason === 'clickaway') {
+            return;
+        }
+        setShowAlert(false);
+        setAlertContent(defaultAlert)
+    };
+
+    useEffect(() => {
+        if (configuration) {
+            methods.setValue('mapping', configuration?.mapping, {shouldDirty: true, shouldTouch: true});
+        }
+        if (configuration?.completed) {
+            setCompleted(true)
+        }
+        if (configuration?.integrationMetadataId
+        ) {
+            getInstanceElementMetadata(configuration.integrationMetadataId.toString())
+        }
+        return () => {
+            resetIntegrationContext()
+            resetConfigurationContext()
+            setEditCollectionAbsoluteKey("")
+            setSelectedMetadata(undefined)
+            setInstanceElementMetadata(undefined)
+        }
+    }, [])
+
+
+    const onSubmit = (data: any) => { // eslint-disable-line
+        if (!isEmpty(methods.formState.errors)) {
+            setAlertContent(errorAlert)
+            setShowAlert(true);
+        }
+        data.mapping = pruneObjectMapping(data.mapping as IObjectMapping)
+        if (configuration?.id) {
+            ConfigurationRepository.updateConfiguration(configuration.id.toString(), data as IConfigurationPatch)
+                .then(response => {
+                    console.log('updated', response)
```

Comment:
Jeg er helt enig i at vi bør har noen clgs for å kunne feilsøke lettere ved behov. Imidlertid ønsker jeg å sette opp lintinga til å markere de så de blir lettere å fjerne senere - eventuelt om vi lager en ticket der vi søker opp alle console.log og fjerner de.

Reply:
enig i det forsåvidt, jeg tenker vi kan fjerne de fleste sjøl. Egen ticket hørtes ut som en god ide ✅